SECURITY.md

```diff
@@ -0,0 +1,23 @@
+# Security Policy
+
+## Supported Versions
+
+Use this section to tell people about which versions of your project are
+currently being supported with security updates.
+
+| Version | Supported          |
+| ------- | ------------------ |
+| 2.5.x   | :white_check_mark: |
+| < 2.5.0 | :x:                |
+| < 1.x.x | :x:                |
```

Comment:
With a small team of maintainers on a niche gem, I think it's reasonable to only support security updates on the latest version. Could save us the hassle of updating the table with every release if we just put that in writing.